vulcan/db/migrations/006_add_workflow_snapshots.rb

```diff
@@ -0,0 +1,29 @@
+Sequel.migration do
+  up do
+    create_table(:workflow_snapshots) do
+      primary_key :id
+
+      foreign_key :figure_id, :figures, null: false, unique: true
+      json :authors, null: false
+      String :name, null: false
+      json :projects
+      String :vignette, text: true
+      String :display_name
+      String :description
+      String :cwl_version, null: false, default: "v1.1"
+      json :inputs
+      json :outputs
+      json :steps, null: false
+      json :other_metadata
```

Comment:
This currently stores non-CWL-type stuff, like the params used by the Query page to render the button. Seems like a catchall, could model it differently.

Reply:
So, to be clear -- are we planning to use this field immediately for the query page render button?  Should we rename this field to reflect that?  I feel like we'd want to avoid any 'catch all' objects and simply explicitly extend the model to meet new needs so that it is more clear what the data actually is.

Reply:
Okay, sure I can rename it. It is currently used for the query page render metadata.